src/build.lock

```diff
@@ -90,35 +90,52 @@ dependencies:
     AirportItlwm-BigSur:
       build: DEBUG
       version: 2.3.0
-      url: 'https://github.com/OpenIntelWireless/itlwm/releases/download/v2.3.0-alpha/AirportItlwm-Big_Sur-v2.3.0-DEBUG-alpha-3e1624d.zip'
-      resolution: 'OpenIntelWireless/itlwm@github:2.3.0#commit=3e1624df2168594ac15bbfe539c751deeedf24cb'
+      url: 'https://github.com/OpenIntelWireless/itlwm/releases/download/v2.3.0-alpha/AirportItlwm-Big_Sur-v2.3.0-DEBUG-alpha-e886ebb.zip'
+      resolution: 'OpenIntelWireless/itlwm@github:2.3.0#commit=e886ebb06658d6aee70f87e30a6bbbd0acad1f57'
       specifier: 'OpenIntelWireless/itlwm@github:^2.0.0'
       kind: Kext
-      revision: '{ SHA1: 3e1624df2168594ac15bbfe539c751deeedf24cb }'
+      revision: '{ SHA1: e886ebb06658d6aee70f87e30a6bbbd0acad1f57 }'
     AirportItlwm-Catalina:
       build: DEBUG
       version: 2.3.0
-      url: 'https://github.com/OpenIntelWireless/itlwm/releases/download/v2.3.0-alpha/AirportItlwm-Catalina-v2.3.0-DEBUG-alpha-3e1624d.zip'
-      resolution: 'OpenIntelWireless/itlwm@github:2.3.0#commit=3e1624df2168594ac15bbfe539c751deeedf24cb'
+      url: 'https://github.com/OpenIntelWireless/itlwm/releases/download/v2.3.0-alpha/AirportItlwm-Catalina-v2.3.0-DEBUG-alpha-e886ebb.zip'
+      resolution: 'OpenIntelWireless/itlwm@github:2.3.0#commit=e886ebb06658d6aee70f87e30a6bbbd0acad1f57'
       specifier: 'OpenIntelWireless/itlwm@github:^2.0.0'
       kind: Kext
-      revision: '{ SHA1: 3e1624df2168594ac15bbfe539c751deeedf24cb }'
+      revision: '{ SHA1: e886ebb06658d6aee70f87e30a6bbbd0acad1f57 }'
     AirportItlwm-Monterey:
       build: DEBUG
       version: 2.3.0
-      url: 'https://github.com/OpenIntelWireless/itlwm/releases/download/v2.3.0-alpha/AirportItlwm-Monterey-v2.3.0-DEBUG-alpha-3e1624d.zip'
-      resolution: 'OpenIntelWireless/itlwm@github:2.3.0#commit=3e1624df2168594ac15bbfe539c751deeedf24cb'
+      url: 'https://github.com/OpenIntelWireless/itlwm/releases/download/v2.3.0-alpha/AirportItlwm-Monterey-v2.3.0-DEBUG-alpha-e886ebb.zip'
+      resolution: 'OpenIntelWireless/itlwm@github:2.3.0#commit=e886ebb06658d6aee70f87e30a6bbbd0acad1f57'
       specifier: 'OpenIntelWireless/itlwm@github:^2.0.0'
       kind: Kext
-      revision: '{ SHA1: 3e1624df2168594ac15bbfe539c751deeedf24cb }'
+      revision: '{ SHA1: e886ebb06658d6aee70f87e30a6bbbd0acad1f57 }'
+    AirportItlwm-Sonoma-14:
+      0:
```

Comment:
The current version of OCE-Build's lock file parser does not handle keys containing periods, which breaks using '14.0' and '14.4' as part of the key names for AirportItlwm.

This creates two entries under `AirportItwlm-Sonoma-14 > 0` and `AirportItwlm-Sonoma-14 > 4` instead.